### ISSUE TITLE
90% there. No Groups show up under group settings.

### ISSUE THREAD

**Issue by themak1985** [veda-consulting/uk.co.vedaconsulting.googlegroup#14]
Just discovered this gem of an extension - we are all setup and got Oauth to work - but I don't see a list of groups.

I did find this in the log:
`[info] Deprecated: Group='Googlegroup Preferences'. Name='access_token'. Defaults should come from metadata`

![Annotation 2020-02-19 172643](https://user-images.githubusercontent.com/28904266/74882494-383f3b80-533d-11ea-9418-427806c4bc18.png)

We do have some funds to support this.

**Comment by veda-consulting** [comment]
@themak1985 hey are you still looking for support on this? We are about to pick up the issue queue and can prioritise accordingly.

**Comment by themak1985** [comment]
Yes - I think it can be fixed by completely deleting the settings - but it always seems to be connected - even when I uninstall.
